### PR TITLE
First version of a PR template, mentioning DCO

### PR DESCRIPTION
**Pull request by axelsimon**
Also mentions reminders of other niceties, such as referencing issues or avoiding merge commits ;)

Closes enarx/enarx#775.

Signed-off-by: axelsimon <github@axelsimon.net>

### CODE REVIEW

**Comment by axelsimon** [comment]
Master of the automation, king of the 'hub, @mbestavros, could you review this? thanks!

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- 
+Thanks for opening a pull request and helping Enarx grow.
+Please remember to
+- mention any issue this PR closes using its number #123 or enarx/repo-name#123, cf.
+  [documentation](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests)
```
Comment (mbestavros) [inline pull_request_template.md:5]: We want more than this: specifically, if this PR closes an issue, it should be linked to that issue with a closing keyword. Just linking to the issue isn't strong enough; it must have a keyword in front of it.

See [here](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for documentation on it.

**Review by mbestavros — CHANGES_REQUESTED** [review]
I have been summoned!

File: pull_request_template.md (hunk 1 above)
Comment (axelsimon) [reply pull_request_template.md:5]: Ah good point. Let me clarify this, thanks. Also, thanks for the link to the documentation.

**Comment by axelsimon** [comment]
I've addressed @mbestavros comments, it looks better now.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!-- 
+Thanks for opening a pull request and helping improve Enarx.
+
+Please remember to:
+- mention any issue(s) that this PR closes using a closing keyword as well as the issue number, such as #123 or enarx/repo-name#123, cf.
```
Comment (mbestavros) [inline pull_request_template.md:5]:
```suggestion
- mention any issue(s) that this PR closes using a closing keyword as well as the issue number, such as "Closes #123" or "Resolves enarx/repo-name#123", cf.
```

**Review by mbestavros — CHANGES_REQUESTED** [review]

**Review by mbestavros — CHANGES_REQUESTED** [review]
Squash this into one commit, and I think it's ready to go!

**Comment by axelsimon** [comment]
Done. Should be good now.

**Review by connorkuehl — APPROVED** [review]

**Review by lkatalin — APPROVED** [review]

**Review by mbestavros — APPROVED** [review]

**Pull request merged**